core/templates/vector.h

```diff
@@ -84,13 +84,13 @@ class Vector {
 	_FORCE_INLINE_ void clear() { resize(0); }
 	_FORCE_INLINE_ bool is_empty() const { return _cowdata.is_empty(); }
 
-	_FORCE_INLINE_ T get(int p_index) { return _cowdata.get(p_index); }
-	_FORCE_INLINE_ const T &get(int p_index) const { return _cowdata.get(p_index); }
-	_FORCE_INLINE_ void set(int p_index, const T &p_elem) { _cowdata.set(p_index, p_elem); }
-	_FORCE_INLINE_ int size() const { return _cowdata.size(); }
+	_FORCE_INLINE_ T get(vec_size p_index) { return _cowdata.get(p_index); }
+	_FORCE_INLINE_ const T &get(vec_size p_index) const { return _cowdata.get(p_index); }
+	_FORCE_INLINE_ void set(vec_size p_index, const T &p_elem) { _cowdata.set(p_index, p_elem); }
+	_FORCE_INLINE_ vec_size size() const { return _cowdata.size(); }
 	Error resize(int p_size) { return _cowdata.resize(p_size); }
 	Error resize_zeroed(int p_size) { return _cowdata.template resize<true>(p_size); }
-	_FORCE_INLINE_ const T &operator[](int p_index) const { return _cowdata.get(p_index); }
+	_FORCE_INLINE_ const T &operator[](vec_size p_index) const { return _cowdata.get(p_index); }
 	Error insert(int p_pos, T p_val) { return _cowdata.insert(p_pos, p_val); }
 	int find(const T &p_val, int p_from = 0) const { return _cowdata.find(p_val, p_from); }
```

Comment:
This is WIP most likely, thought beware that `find` should return `uint32_t` too. Usually other libs define `INDEX_NONE = UINT32_MAX` that is possible to use to specify unknowns positions.

```
if( my_strings.find("TheText") == INDEX_NONE )
```

----
Note, `local_vector.h` returns `int64_t`, so return `-1` is not an undefined behavior; though from user prospective it's annoying convert `int64_t` to `uint32_t`, it usually requite an extra if condition - so I find much comfortable the `INDEX_NONE` solution.